### PR TITLE
add snabbdom@0.5.4 w/ npm auto-update

### LINKED ISSUE #9785: [Request] Add snabbdom
**Library name:** snabbdom
**Git repository url:** https://github.com/snabbdom/snabbdom
**npm package url(optional):** https://www.npmjs.com/package/snabbdom
**License(s):** MIT
**Official homepage:**

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #9785
Related issue(s): # #9414

@kennynaoh please check this PR, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/snabbdom/snabbdom
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/snabbdom
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1879
   - Count of watchers: 77
   - Count of forks: 126
 * NPM download stats (optional):
   - Downloads in the last day: 249
   - Downloads in the last week: 1042
   - Downloads in the last month: 4764

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9785, cc @paldepind

### CODE REVIEW

**Comment by hare1039** [comment]
@kennynaoh I've switch to npm auto-update. Please review.

**Comment by PeterDaveHello** [comment]
@hare1039 Please also update the PR title.

**Comment by hare1039** [comment]
PR title updated. Please check.

File: ajax/libs/snabbdom/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "snabbdom",
+  "version": "0.5.4",
+  "description": "A virtual DOM library with focus on simplicity, modularity, powerful features and performance.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/paldepind/snabbdom.git"
```
Comment (kennynaoh) [inline ajax/libs/snabbdom/package.json]: @hare1039 I think it's better to use `git+https://github.com/snabbdom/snabbdom.git` here.

**Comment by hare1039** [comment]
@kennynaoh Ok, I've change the url to `git+https://github.com/snabbdom/snabbdom.git`

File: ajax/libs/snabbdom/package.json (hunk 1 above)
Comment (hare1039) [inline ajax/libs/snabbdom/package.json]: @kennynaoh Ok, I've change the url to `git+https://github.com/snabbdom/snabbdom.git`

**Comment by kennynaoh** [comment]
@hare1039 Please cc the related issue #9414 in commit message.

**Comment by hare1039** [comment]
@hare1039 Amended the commit massage. Please check.

**Comment by hare1039** [comment]
@kennynaoh Amended the commit massage. Please check.

**Comment by kennynaoh** [comment]
@hare1039 I think it's better to use review instead of check next time, thanks!

**Comment by kennynaoh** [comment]
LGTM
ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**